psi4/driver/procrouting/sapt/sapt_jk_terms.py

```diff
@@ -292,56 +292,55 @@ def induction(cache, jk, do_print=True, maxiter=12, conv=1.e-8, do_response=True
 
     # Exch-Ind Potential A
     EX_A = K_B.clone()
-    EX_A.scale(-1.0)
```

Comment:
There are a lot of terms changed here which presumable yield the same thing. Can you comment on why the ordering was changed?

Reply:
The ordering is changed to fit the formula in Jerome Gonthier's note on USAPT0, hoping that anyone who reads the code would be less confused. They should be in the CDSgroup/notes repo. 

I am also rephrasing the equations into my SAPT(DFT) notes while rephrasing them in terms restricted orbitals and density matrices.